root/static/scripts/common/components/EditorLink.js

```diff
@@ -23,7 +23,7 @@ const MissingEditorLink = () => {
 type Props = {|
   +avatarSize?: number,
   +content?: string,
-  +editor: EditorT | SanitizedEditorT | null,
+  +editor: ?(EditorT | SanitizedEditorT),
```

Comment:
I think you accidentally reverted some of the type changes I synced here, this and the `AnnotationT` change in root/types.js